docs/components/Counter.md

```diff
@@ -4,25 +4,25 @@ The Counter component is used for when it's necessary to count something. For ex
 
 ```js
 import { Counter } from 'react-powerplug'
-``` 
+```
 
 ```jsx
 <Counter initial={0}>
   {({ count, inc, dec }) => (
     <CartItem
       productName="Lorem ipsum"
-      unitPrice={19.90}
+      unitPrice={19.9}
```

Comment:
I think we should keep the trailing zero here.

Reply:
Why do you think so?

Reply:
Because semantically it is more correct, this example portrays a "cart" with a unit "price".

When you go to Starbucks the price will be with trailing zero `$3.90`, not `$3.9`.

Reply:
Well, it's javascript, not a string. And if you care about correctness, comma is more correct here `$3,90`

Reply:
Anyway docs are formatted with prettier.

Reply:
That's cool, I did not know Prettier does that, will try it out.